meson.build

```diff
@@ -0,0 +1,12 @@
+project('squashfs-mount', ['cpp'],
```

Comment:
Change the name of the project to `slurm-uenv-mount`?

Reply:
meson.build was removed in a PR of mine. This branch should rebase with master first.

Reply:
We should move meson.build to another directory if we want to keep it in the repo at all. Otherwise it suggests it is actively used. However, since it isn't used we should remove it. It is still in the history if it is needed again and it isn't very complicated in the first place.

Reply:
Ok, I've removed the meson.build. But I think it wasn't a problem. One can use either meson or the Makefile.